src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-tower-integration.mdx

```diff
@@ -0,0 +1,150 @@
+
+
+## Get Ansible Tower metrics in a dashboard [#dash]
+
+Our Ansible Tower integration builds atop our Prometheus remote write integration. To view your data in our quickstart dashboard, follow the below steps to integrate Ansible Tower, Prometheus, and the New Relic platform. 
+
+<CollapserGroup>
+<Collapser
+    id="logging"
+    title="Step 1. Connect Ansible Tower logs to the infrastructure agent"
+  >
+Navigate `/etc/newrelic-infra/logging.d/`. If you don't see a `logging.yml` file, then create one.
+
+Add this script to the `logging.yml` file:
+
+```yml
+- name: ansible-tower.log
+  file: /var/log/tower/tower.log
+  attributes:
+    logtype: ansible_tower_log
+```
+</Collapser>
+<Collapser
+  id="integrate"
+  title="Step 2. Integrate Prometheus in the New Relic platform"
+  >
+From your Prometheus folder, run `./prometheus` to start your server.  In the `prometheus.yml` file, add this script:
+
+```yml
+remote_write:
+  - url: "YOUR URL"
+    bearer_token: "YOUR TOKEN"
+scrape_configs:
+  - job_name: "prometheus"
+  static_configs:
+    - targets: ["localhost:9100"]
+```
+
+This script: 
+
+* Connects Prometheus to the New Relic platform via `remote_write`. 
+* Sets `"prometheus"` as the target so it can scrape data from itself.
+</Collapser>
+<Collapser
+  id="jobs"
+  title="Step 3. Grab Ansible Tower metrics"
+>
+
+With data from Prometheus exposed to New Relic, you can now integrate your Ansible Tower projects into the New Relic platform. 
+
+Add this script to that same  `prometheus.yml` file: 
+
+  ```yml
+- job_name: "tower"
+    tls_config: 
+      insecure_skip_verify: true
+    metrics_path: /api/v2/metrics
+    scrape_interval: 5s
+    scheme: https
+    bearer_token: "YOUR_TOKEN"
+    static_configs: 
+      - targets:  ["localhost:443"]
+- job name: "node"
+    static_configs: 
+      targets: ['localhost:9100']
+```
+
+This script:
+
+* Puts Ansible Tower as the target to scrape data from.
+* Sets a metric path to pull that scraped data.
+* Defines the scrape interval.
+ 
+</Collapser>
+<Collapser
+
+  id="nrql"
+  title="Step 4. Build NRQL queries to generate more dashboards"
+>
+With your data ingested and now displaying in your dashboard, you can customize the visualizations based on your team's needs. New Relic dashboards are built and generated with NRQL queries.
+
+To get this dashboard, for example:
+
+<img
+  title="queryansibleexample.png"
+  alt="queryansibleexample.png"
+  src={queryAnsibleExample}
+/>
+
+You would use this query:
+
+```sql
+FROM Metric SELECT count(status) FACET status SINCE 7 days ago WHERE prometheus_server = 'prometheus_remote' AND job = 'tower'
+```
+</Collapser>
+</CollapserGroup>
+
+Now you can view your data with your new Ansible Tower dashboard. After navigating to New Relic, find the **Dashboards** tile in the top nav. 
```

Comment:
@akristen this seems like it should appear first (is the cart before the horse?). If you've already asked users to create custom queries, they would do it from dashboards, no?